protocol/contracts/beanstalk/silo/SiloFacet/TokenSilo.sol

```diff
@@ -94,6 +94,22 @@ contract TokenSilo is Silo {
         uint256 amount,
         uint256[] bdvs
     ); //add bdv[] here? in favor of array
+    
+    // per the zero withdraw update, there is no claiming function for withdraws.abi
+    // events are kept for backwards compatibility
```

Comment:
I thought we decided there could be some in-flight withdrawals that have not yet been claimed, so those claim functions need to be kept accessible? or is this comment just referring to the exact abi file where those will be visible?